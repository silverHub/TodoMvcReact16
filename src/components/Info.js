import './Info.css';
import React from 'react';

function Info() {
	return (
		<footer className="info">
			<p>Double-click to edit a todo</p>
			<p>Created by Tamas Hoffman</p>
			<p>Inspired by <a href="http://todomvc.com">TodoMVC</a></p>
		</footer>
	);
}

export default Info;
