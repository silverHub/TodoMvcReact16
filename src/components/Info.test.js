import Info from './Info';
import React from 'react';
import {shallow} from 'enzyme';

describe('test Info component', () => {
	test('if renders', () => {
		const wrapper = shallow(<Info />);
		expect(wrapper).toMatchSnapshot();
	});
});



