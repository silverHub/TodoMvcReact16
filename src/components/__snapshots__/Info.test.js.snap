// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test with react-test-renderer if renders, snapshot 1`] = `
<footer
  className="info"
>
  <Input />
  <p>
    Double-click to edit a todo
  </p>
  <p>
    Created by Iurii Bura, Konrad Hefelle, Tamas Hoffman
  </p>
  <p>
    Inspired by 
    <a
      href="http://todomvc.com"
    >
      TodoMVC
    </a>
  </p>
</footer>
`;
